pkgs/build-support/setup-hooks/writable-tmpdir-as-home.sh

```diff
@@ -0,0 +1,15 @@
+# shellcheck shell=bash
+# This setup hook set the HOME environment variable to a writable directory.
+
+export HOME
+
+writableTmpDirAsHome () {
```

Comment:
If we name the hook "writable" shouldn't it also call chmod +w or so to make sure it is writable? Right now with the wrong umask it isn't

Reply:
It wasn't needed so far, but it could be done in a future PR.